doc/classes/Input.xml

```diff
@@ -343,6 +343,20 @@
 				[b]Note:[/b] This value can be immediately overwritten by the hardware sensor value on Android and iOS.
 			</description>
 		</method>
+		<method name="should_ignore_device" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="vendor_id" type="int" />
+			<param index="1" name="product_id" type="int" />
+			<description>
+				Returns whether the device is requested to be ignored or not.
+
+				There are certain scenarios where game devices should be ignored; The most common scenario being that a 3rd party api (for example, [i]SteamInput[/i]) is already using an existing
+				device to add a new virtual device which should be watched instead. In such a case, we should respect the virtual device and ignore the original device that is being translated.
+				This allows for full gamepad rebinding features without resulting in multiple input events being registered from the same device.
+
+				Devices can be ignored by setting the environment variable [code]SDL_GAMECONTROLLER_IGNORE_DEVICES[/code] before booting the Godot Engine or Project. Read SDL documentation for more details.
```

Comment:
There shouldn't be any empty lines in the description, or line breaks mid-sentence. Each new line creates a new paragraph.

```suggestion
				Returns whether the device is requested to be ignored or not. Devices can be ignored by setting the [code]SDL_GAMECONTROLLER_IGNORE_DEVICES[/code] environment variable before booting Godot or your project. Read SDL documentation for more details.
				[b]Note:[/b] Some 3rd party tools can contribute to the list of ignored devices. For example, [i]SteamInput[/i] creates virtual devices to allow free key and input remapping. To avoid handling the same input device twice, the original device is added to the ignore list.
```

`Read SDL documentation for more details.` should also link to the said documentation.

Reply:
Thanks for the review. 

I've gone ahead and fixed the formatting. Additionally, I did some rewording to make it easer to read and be a bit more concise. Let me know if there's any other changes you'd like!